Mathlib/Analysis/NormedSpace/IndicatorFunction.lean

```diff
@@ -36,9 +36,21 @@ theorem norm_indicator_le_of_subset (h : s ⊆ t) (f : α → E) (a : α) :
   simp only [norm_indicator_eq_indicator_norm]
   exact indicator_le_indicator_of_subset ‹_› (fun _ => norm_nonneg _) _
 
+theorem enorm_indicator_le_of_subset (h : s ⊆ t) (f : α → E) (a : α) :
```

Comment:
generalize codomain? (also below, if possible). Oh wait, this does require `enorm 0 = 0`, then this has to wait until more classes being ported.